{
 "bad": [
  "py.path.local",
  "pytest.ExitCode.OK",
  "pytest.ExitCode.TESTS_FAILED",
  "sys",
  "sys.pypy_version_info",
  "time"
 ],
 "deps": {
  "__future__": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "_pytest": [],
  "platform": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "pluggy": [
   "pluggy/conda-forge/linux-64/pluggy-0.13.1-py38h578d9bd_4",
   "pluggy/conda-forge/noarch/pluggy-0.11.0-py_0",
   "pluggy/conda-forge/noarch/pluggy-0.12.0-py_0",
   "pluggy/conda-forge/noarch/pluggy-0.6.0-py_0"
  ],
  "py": [
   "py/conda-forge/noarch/py-1.5.1-py_0",
   "py/conda-forge/noarch/py-1.5.2-py_0",
   "py/conda-forge/noarch/py-1.5.3-py_0",
   "py/conda-forge/noarch/py-1.6.0-py_0",
   "py/conda-forge/noarch/py-1.7.0-py_0",
   "py/conda-forge/noarch/py-1.8.1-py_0",
   "py/conda-forge/noarch/py-1.8.2-pyh9f0ad1d_0",
   "py/conda-forge/noarch/py-1.9.0-pyh9f0ad1d_0"
  ],
  "pytest": [
   "pytest/conda-forge/linux-64/pytest-6.0.2-py37hc8dfbb8_0"
  ]
 }
}